skl2onnx/_parse.py

```diff
@@ -450,6 +452,127 @@ def parse_sklearn(scope, model, inputs, custom_parsers=None, final_types=None):
     return outputs
 
 
+def parse_sklearn(scope, model, inputs, custom_parsers=None, final_types=None):
+    """
+    This is a delegate function. It does nothing but invokes the
+    correct parsing function according to the input model's type.
```

Comment:
Both of them are the delegate function? I saw this comment twice?

Reply:
Very old comment. I need to check.